### DISCUSSION TITLE
Help with shaders (EGA)

### DISCUSSION THREAD

**Discussion by T-Lari** [dosbox-staging/dosbox-staging#3586] [General]
So, I'm kinda new with this. I want to use an EGA shader on some VGA games, but I'm not being sucessful this far. I'm not sure if it's due to my computer being very old or if I'm doing something wrong.

I'm using the DBGL frontend. First I tried to use a GLSL file I found here: https://www.vogons.org/viewtopic.php?p=289834#p289834

I tried to load it on DOSBox-Staging. But it doesn't show anything. There's a HSLS there as well but I don't know how it works, so I tried a .fx file that comes on DOSBox-SVN-Daum. It didn't worked as well. Anyone can help me setting this on DBGL? Or show where I can find a good EGA shader file? I don't mind using any Dosbox version for that, I just want it to work slightly frowning face

Here's my PC specs:
Inter(R) Core(TM) i5-2310 CPU @ 2.90GHz 2.90GHz
RAM 6,00 GB
Windows 7 Ultimate, Service Pack 1, 64 Bits
DirectX 9, 10.1
- Shader model: vs_4_1, ps_4_1
OpenGL 3.1
- Driver version: 9.17.10.4229 (.dll)
Vulkan
- Not available

Thank you very much in advance and sorry for the annoyance

**Comment by johnnovak** [comment]
@T-Lari DOSBox Staging v0.81.0 (our latest release) comes with authentic PC CRT monitor emulation out-of-the-box. You'll get authentic EGA emulation in EGA games with the default config—no need to configure anything or tinker with shaders manually!

See our front page for example screenshots:
https://dosbox-staging.github.io/

Our Getting Started guide also explains the various CRT emulation options in detail:
https://dosbox-staging.github.io/getting-started/introduction/

Have fun 😎
